test/io/test_iterators.cpp

```diff
@@ -59,9 +59,9 @@ TEST (PCL, Iterators)
 {
   Point mean (0,0,0);
 
-  for (auto it = cloud.begin(); it != cloud.end(); ++it) 
```

Comment:
Since this check seems to test `cloud.begin()` and `cloud.end()`, I would rather keep them visible instead of "hiding" them in a range-for loop